docs/topics/testing-rules.adoc

```diff
@@ -99,7 +99,7 @@ The `<not>` element has no unique attributes or child elements.
 [discrete]
 ===== Summary
 
-The `<iterable-filter>` element counts the number of times a condition is verified. For additional information, see the link:{LinkAPI}org/jboss/windup/rules/general/IterableFilter.html[IterableFilter] class.
+The `<iterable-filter>` element counts the number of times a condition is verified. For additional information, see the link:https://github.com/windup/windup/blob/master/rules-base/api/src/main/java/org/jboss/windup/rules/general/IterableFilter.java[IterableFilter] class.
```

Comment:
Why have you not used the `LinkAPI` attribute for consistency?

Reply:
`Link API` = `https://github.com/windup/windup/blob/master/reporting/api/src/main/java/`
This link begins `https://github.com/windup/windup/blob/master/rules-base`  ... you can see that they diverge after the `master` sub-directory.

This was the last link I found, so I thought it would be safer to leave `LinkAPI` for the 2 xrefs I had verified as is and copy-paste the complete path directory for this one, rather than truncate `LinkAPI` and rewrite the other links from `master`. 

I've now truncated `LinkAPI`, used it for all three links, and adjusted the other two links as needed.

Reply:
Yes you have failed to address the question

You should have shortened the {LINKAPI} attribute or created a second attribute

What happens if the repo is moved again? Then we only have to update one or two attributes

Reply:
@anarnold97 On Tuesday, I truncated {LinkAPI} as you suggested, so that it now can be used with all of the links to the files in the repo. I have squashed my commits, so the changes between versions of the PR aren't visible.